.github/workflows/clang-format-check.yml

```diff
@@ -13,5 +13,11 @@ jobs:
         source: '.'
         extensions: 'c,h,cpp,hpp'
         clangFormatVersion: 10
+        inplace: True
         style: file
         exclude: './config ./hl/src/H5LTanalyze.c ./hl/src/H5LTparse.c ./hl/src/H5LTparse.h ./src/H5Epubgen.h ./src/H5Einit.h ./src/H5Eterm.h ./src/H5Edefin.h ./src/H5version.h ./src/H5overflow.h'
+    - uses: EndBug/add-and-commit@v7
+      with:
+          author_name: github-actions
+          author_email: 41898282+github-actions[bot]@users.noreply.github.com
+          message: 'Committing clang-format changes'
```

Comment:
Have these changes already made it into develop? This looks like it should be in a separate PR.

Reply:
Working on it here - needs trial runs.

Reply:
If we're making the change in the 1.10 branch, ok. I just thought we were going to use develop instead to test it.

Reply:
I needed to test it - and this was an active development for me.